server/tests/auth/conftest.py

```
@@ -0,0 +1,78 @@
+
+@pytest.fixture
+def app():
+    """Minimal Flask app with the incidents blueprint registered.
+
+    Force-evicts and re-imports the route module and its Flask-dependent
+    transitive dependencies on every fixture instantiation, so the
+    Werkzeug LocalProxy objects inside them are bound to the Flask instance
+    created in *this* test run rather than one from a prior test file.
+    """
+    for _mod in list(sys.modules):
+        if _mod.startswith("routes.") or _mod.startswith("utils.auth.rbac"):
```

Ruff (0.15.12)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)